docs/FATE_On_Spark_With_Pulsar.md

```diff
@@ -70,6 +70,7 @@ When submitting a task, the user can declare in the config file to use Pulsar as
    "job_parameters": {
      "common": {
        "job_type": "train",
+       "backend": 2,
```

Comment:
Seems after FATE 1.7, we don't need to specify backend in the job parameter. I'll let @owlet42 chim in on this.

Reply:
Fixed, please help review again